--- a/binder.c
+++ b/binder.c
@@ -7164,3 +7164,6 @@
 
 #define CREATE_TRACE_POINTS
 #include "binder_trace.h"
+
+MODULE_LICENSE("GPL v2");
+MODULE_DESCRIPTION("Android Binder IPC Driver");
--- a/binder_alloc.c
+++ b/binder_alloc.c
@@ -39,7 +39,7 @@ enum {
 };
 static uint32_t binder_alloc_debug_mask = BINDER_DEBUG_USER_ERROR;
 
-module_param_named(debug_mask, binder_alloc_debug_mask,
+module_param_named(alloc_debug_mask, binder_alloc_debug_mask,
 		   uint, 0644);
 
 #define binder_alloc_debug(mask, x...) \
